pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
`-Dfindbugs.failOnError=false` is set on the command line by the template on jenkins.ci.cb.com so for the CI builds I still get what I want :) So it doesn't matter to me what the setting is default in the pom.